Add uuid to access token

iplonk3 opened the pull request.

Commit:
Add uuid to access token

Commit:
Include specific classes imported

Commit:
Remove extra space

Castyr commented on src/main/java/org/aitesting/microservices/authentication/security/JWTTokenEnhancer.java:
```diff
@@ -17,15 +20,25 @@
 
+    private UUID getUserId(String username) {
+        User user = userRepo.findByUsername(username);
+        return user.getUserId();
```
What does this return if the user is not found? It looks like an exception could be thrown by this code.

jusoto committed:
Merge branch 'master' of https://github.com/AITestingOrg/authentication-service into connect-frontend-to-auth

jusoto committed:
Added:
- Logger in JWTTokenEnhancer and check objects not null before reading properties
- Changed Eureka Server environment variables name to EUREKA_SERVER

jusoto committed:
added environment variable DOCKER_IP for test application.yml needed on Windows with Docker Toolbox and localhost for Docker
